app/models/book.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class Book < ApplicationRecord
+  validates :genre, presence: true
```

Comment:
Maybe [Multiple attributes validations](https://apidock.com/rails/ActiveModel/Validations/ClassMethods/validates#925-multiple-attributes-with-the-same-validations) can improve this code. Remember [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).

Reply:
didnt remember that i can do it in just one line